source/jormungandr/jormungandr/realtime_place/bss/bss_provider_manager.py

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) 2001-2014, Canal TP and/or its affiliates. All rights reserved.
+#
+# This file is part of Navitia,
+#     the software to build cool stuff with public transport.
+#
+# Hope you'll enjoy and contribute to this project,
+#     powered by Canal TP (www.canaltp.fr).
+# Help us simplify mobility and open public transport:
+#     a non ending quest to the responsive locomotion way of traveling!
+#
+# LICENCE: This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+# Stay tuned using
+# twitter @navitia
+# IRC #navitia on freenode
+# https://groups.google.com/d/forum/navitia
+# www.navitia.io
+from jormungandr import app
+from importlib import import_module
+import logging
+
+
+class BssProviderManager(object):
+
+    def __init__(self):
+        self.instances = []
+        self.log = logging.getLogger(__name__)
+        for configuration in app.config["BSS_PROVIDER"]:
+            arguments = configuration.get('args', {})
+            self.instances.append(self.init_class(configuration["class"], arguments))
+
+    def handle_places(self, places):
+        if places:
+            for place in places:
+                if (place["embedded_type"] == "poi") & (place["poi"]["poi_type"]["id"] == 'poi_type:amenity:bicycle_rental'):
```

Comment:
Parenthesis aren't required, for a logical and you should use ```and```
And by the way the preferred style for string is simple quote.